cmd/juju/model/defaults.go

```diff
@@ -151,6 +227,7 @@ func (c *defaultsCommand) SetFlags(f *gnuflag.FlagSet) {
 		"tabular": formatDefaultConfigTabular,
 	})
 	f.Var(cmd.NewAppendStringsValue(&c.reset), "reset", "Reset the provided comma delimited keys")
+	f.BoolVar(&c.skipImmutableErrors, "skip-immutable-errors", false, "Skip immutable errors when passing in the configurations")
```

Comment:
I think the help text and perhaps the flag itself might need rephrasing to make it easier for end-users to understand what it actually does (I _bet_ people would be all over SO asking what an immutable error is).  Perhaps something like 'ignore-read-only-fields' "ignore changes to read-only fields instead of reporting an error" would be clearer.